packages/storage/README.md

```diff
@@ -48,6 +48,11 @@ remoteReadStream.pipe(localWriteStream);
 var localReadStream = fs.createReadStream('/photos/zoo/zebra.jpg');
 var remoteWriteStream = bucket.file('zebra.jpg').createWriteStream();
 localReadStream.pipe(remoteWriteStream);
+
+// Promises are also supported by omitting callbacks.
+bucket.upload('/photos/zoo/zebra.jpg').then(function(data) {
+  var file = data[0];
+});
```

Comment:
Is it possible and/or necessary for the user to understand how our API works to change the JSDocs? Since the demand for promises is so strong, I'm wondering if we might be better off changing our position to a promise library that supports callbacks instead of the other way around.